commit f2efa15ef8c083926791195ed1037fd47a5e4e41
Author: Scott Mayhew <smayhew@redhat.com>
Date:   Wed Sep 16 11:19:41 2015 -0400

    mountstats.man:  Remove a few bogus .R macros
    
    These have no effect on the rendering of the man page, but they do cause
    the following error if you try to pipe or redirect the output:
    
    `R' is a string (producing the registered sign), not a macro.
    
    Signed-off-by: Scott Mayhew <smayhew@redhat.com>
    Signed-off-by: Steve Dickson <steved@redhat.com>

diff --git a/tools/mountstats/mountstats.man b/tools/mountstats/mountstats.man
index bfc4c46..a55130b 100644
--- a/tools/mountstats/mountstats.man
+++ b/tools/mountstats/mountstats.man
@@ -13,13 +13,13 @@ mountstats \- Displays various NFS client per-mount statistics
 .RB [ \-S | \-\-since
 .IR sincefile ]
 .\" .RB [ \-n | \-\-nfs | \-r | \-\-rpc | \-R | \-\-raw ]
-.R [
+[
 .RB [ \-n | \-\-nfs ]
-.R |
+|
 .RB [ \-r | \-\-rpc ]
-.R |
+|
 .RB  [ \-R | \-\-raw ]
-.R ]
+]
 .RI [ mountpoint ] ...
 .P
 .B mountstats iostat
